WordPress/Classes/ViewRelated/Post/PostSettingsViewController+FeaturedImageUpload.swift

```diff
@@ -13,6 +14,7 @@ extension PostSettingsViewController {
             apost.featuredImage = media
             return
         }
+        apost.featuredImage = media
```

Comment:
I guess we don't need to duplicate this line in here – if we move it to the top of the method, we can remove one of the instances:

```
apost.featuredImage = media

if !media.hasRemote {
    MediaCoordinator...
    setupObservingOf...
}
```